### ISSUE TITLE
Always pass youtube url to player

### ISSUE THREAD

**Issue by Slider-Whistle** [trizen/pipe-viewer#27]
Hi, I don't understand enough about Perl to figure this out myself, but I think a setting or a new *VARIABLE\* to always pass "youtube_video_url" to the selected player, even when pipe-viewer is falling back to the Invidious API to grab search results and comments would be nice.
Invidious nodes tend to be a little slower, and plugin support in the youtube-dl project (I removed the --no-ytdl arg in my config) for it is understandably slightly less maintained than its support for youtube itself. In my case mpv's "ytdl-format" setting isn't working as-is, and I think the reason might be that youtube-dl doesn't know how to select video resolutions with Invidious.
pipe-viewer's "-2", "-3" arguments do work, and I saw something to do with resolution in the config file, but I'd still enjoy the faster speed of watching videos from youtube itself, and unlike the rest of the API, that point will only ever fail if you get a http 503.

**Comment by trizen** [comment]
Hi. There is the `*URL*` special token than can be used for passing the YouTube URL to a video player.

See also: https://github.com/trizen/pipe-viewer/issues/13

**Comment by Slider-Whistle** [comment]
Whoops! Sorry.

**Comment by Slider-Whistle** [comment]
I realise I'm posting on a closed issue, but I finally figured out what
seemed so different between loading a video through pipe-viewer and
through mpv by itself. When loaded through pipe-viewer, audio seems to
become momentarily desynced with the video when seeking, even being mute
for a second or two. This is because I had pipe-viewer configured to
force its own audio stream into mpv (is this default behaviour? I
wouldn't really be able to tell). By commenting out mpv's "audio"
argument in .config/pipe-viewer.conf, I was able to fix this. Now it's
indistinguishable to loading a youtube video URL in mpv directly.
